README.md

```diff
@@ -25,6 +25,13 @@ https://open-dollar-app.vercel.app/
 ## Development
 
 ### Install Dependencies
+```bash
```

Comment:
This should be moved to the run category. You cannot build until after deps have been installed